RUFAS/biophysical/animal/animal_module_constants.py

```diff
@@ -79,9 +79,13 @@ class AnimalModuleConstants:
     MILK_LACTOSE: float = 4.85
     """Milk lactose content, percentage."""
 
-    DMI_CONSTRAINT_FRACTION: float = 0.20
+    DMI_CONSTRAINT_FRACTION: float = 0.30
     """The +/- fraction of DMI estimated allowed for ration formulation."""
 
+    DMI_requirement_boost: float = 1.1
```

Comment:
```suggestion
    DMI_REQUIREMENT_BOOST: float = 1.1
```
Just to match all the other constants